core/src/main/java/pl/senla/hotel/comparators/HotelServicesComparatorByPrice.java

```diff
@@ -1,25 +1,26 @@
 package pl.senla.hotel.comparators;
 
 import lombok.extern.slf4j.Slf4j;
-import pl.senla.hotel.application.annotation.AppComponent;
-import pl.senla.hotel.application.annotation.GetInstance;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import pl.senla.hotel.dao.DaoHotelFacilitySpring;
 import pl.senla.hotel.entity.facilities.Room;
 import pl.senla.hotel.entity.services.HotelService;
-import pl.senla.hotel.dao.GenericDao;
 import pl.senla.hotel.entity.services.TypeOfService;
 
 import java.lang.reflect.InvocationTargetException;
 import java.util.Comparator;
 import java.util.Objects;
+import java.util.Optional;
 
 import static pl.senla.hotel.constant.HotelServiceConstant.ERROR_IN_SERVICE_TYPE;
 
-@AppComponent
+@Component
 @Slf4j
 public class HotelServicesComparatorByPrice implements Comparator<HotelService> {
```

Comment:
why do you still need this? sort it in db

Reply:
ready